doc/src/tutorial/printing.rst

```diff
@@ -229,6 +229,32 @@ format, which can be rendered with Graphviz.  See the
 :ref:`tutorial-manipulation` section for some examples of the output of this
 printer.
 
+    >>> from sympy.printing.dot import dotprint
+    >>> from sympy.abc import x
+    >>> print(dotprint(x+2)) 
+    digraph{
+
```

Comment:
replace L236, L240, L244, L248, L252 with `<BLANKLINE>`.

I think this would let your test cases pass.